### DISCUSSION TITLE
can the "negotiated_type" be "bundle"?

### DISCUSSION THREAD

**Discussion by nmayle** [CMSgov/price-transparency-guide#574] [Q&A] [answered]
We see negotiated_type with a value of "bundle" in a 08-01 UHC file.  
Is this allowed?

Our reading of the schema is that the bundle is indicated in the negotiation_arrangement in the in-network object not down in the negotiated price object.
Shouldn't the negotiated_type be one of: negotiated, derived, fee schedule, percentage, per diem ?

Here is an example from a file:

```
"negotiated_prices": [
      {
	"negotiated_rate": 23000,
	"service_code": [],
	"negotiated_type": "bundle",
	"expiration_date": "2099-12-31",
	"billing_class": "institutional",
	"billing code_modifier": [],
	"additional information": "days[0-0]"
      }
    ]
```

**Comment by Ginger228** [comment, marked as answer]
Hi- 

Per the schema the negotiated_arrangement may have a value of 'bundle' 
The negotiated_type= should be negotiated, derived, fee schedule, percentage or per diem
See the example posted (https://github.com/CMSgov/price-transparency-guide/blob/master/examples/in-network-rates/in-network-rates-bundle-single-plan-sample.json)
